webpack/ForemanTasks/Components/TasksDashboard/TasksDashboard.scss

```diff
@@ -1,6 +1,27 @@
+
```

Comment:
Does it make sense to move this code into `TasksDashboard/Components/TasksCardsGrid/TasksCardsGrid.scss`?

Reply:
I'm not sure, as it refers to the `tasks-dashboard-grid` class, which is used only in `TasksDashboard.js`